src/components/singleUser/singleUser.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+const SingleUser = ({users}) => {
```

Comment:
this component could be just `UsersBalance`

when choosing the name for your component imagine that somebody will use it in other parts of the application
the component name should describe the output it will produce

So for example the most logical interface would be

```
    <UsersBalance users={....} />
```